app/page.tsx

```diff
@@ -1,9 +1,22 @@
+import Experience from '@/components/Homepage/Experience';
+import Featured from '@/components/Homepage/Featured';
+import Hero from '@/components/Homepage/Hero';
+import Services from '@/components/Homepage/Services';
+import Skills from '@/components/Homepage/Skills';
+import Testimonies from '@/components/Homepage/Testimonies';
 import React from 'react';
 
 const page = () => {
   return (
-    <main className='padding-layout'>
-      <h1>Professional Web Developer based in USA </h1>
+    <main className=''>
+      <section>
```

Comment:
I find it strange to have a section here. Since every one of its child are also sections. You should probably remove it.